docs/installing-cabal.md

```diff
@@ -5,16 +5,17 @@ Cabal comes in two parts, the `cabal` executable (which is in the
 cabal-install package) and the Cabal library (which is in the Cabal
 package).
 
-#### Install cabal-install 1.24.0.0
+#### Install cabal-install 1.24.0.0/2.0.0.1
```

Comment:
Actually can you update that to use 1.24.0.2 instead of 1.24.0.0?

Reply:
Should I replace all instances of `1.24.0.0` to `1.24.0.2` in this doc?

Reply:
Yeah, that would be a good idea.

Reply:
Actually, is it `1.24.2.0`?